specification/gedcom.md

```diff
@@ -2659,6 +2675,12 @@ the authority owning the identifier is provided in the TYPE substructure; see `E
 
 Depending on the maintaining authority, an `EXID` may be a unique identifier for the subject, an identifier for 1 of several views of the subject, or an identifier for the externally-maintained copy of the same information as is contained in this structure. However, unlike `UID` and `REFN`, `EXID` does not identify a structure; structures with the same `EXID` may have originated independently rather than by edits from the same starting point.
 
+:::deprecation
+Having an `EXID` without an `EXID`.`TYPE` substructure is deprecated.
+The meaning of an `EXID` depends on its `EXID`.`TYPE`.
+The cardinality of `EXID`.`TYPE` will be changed to `{1:1}` in version 8.0.
+:::
```

Comment:
I don't think the note is needed here since there's nothing in this section that is otherwise wrong, and line 2673-2674 already implies the TYPE has to exist.  The other two locations look good.

Reply:
I added it here because this is the section that gets auto-extracted as https://gedcom.io/terms/v7/EXID and I wanted it to show up there. But you're right, it looks odd to have a deprecation in the spec that refers to something not local to it.

Reply:
I would maintain that it's not needed there either because it already says "the authority owning the identifier
**is** provided in the TYPE substructure" (i.e., "is provided" not "may be provided")

Reply:
Good point. Fixed